VisualRiders.PointOfSale.Project/Services/StaffMembersService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using AutoMapper;
+using VisualRiders.PointOfSale.Project.DTOs;
+using VisualRiders.PointOfSale.Project.Models;
+using VisualRiders.PointOfSale.Project.Repositories;
+
+namespace VisualRiders.PointOfSale.Project.Services;
+
+public class StaffMembersService
+{
+    private readonly StaffMembersRepository _repository;
+    private readonly BusinessEntitiesService _businessEntitiesService;
+    private readonly IMapper _mapper;
+
+    public StaffMembersService(StaffMembersRepository repository, BusinessEntitiesService businessEntitiesService, IMapper mapper)
+    {
+        _businessEntitiesService = businessEntitiesService;
+        _repository = repository;
+        _mapper = mapper;
+    }
+
+    public ReadStaffMemberDto Create(CreateUpdateStaffMemberDto dto)
+    {
+        var staffMember = _mapper.Map<StaffMember>(dto);
+        if (_businessEntitiesService.GetById(staffMember.BusinessEntityId) == null)
+        {
+            throw new ArgumentException();
```

Comment:
You could have put a meaningful exception message here for use in the controller